<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainViewer</class>
 <widget class="QWidget" name="MainViewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>890</width>
    <height>757</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::StrongFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_3">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QFrame" name="frame">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="FileTreeWidget" name="FileTree">
          <property name="editTriggers">
           <set>QAbstractItemView::DoubleClicked</set>
          </property>
          <property name="dragEnabled">
           <bool>true</bool>
          </property>
          <property name="dragDropMode">
           <enum>QAbstractItemView::InternalMove</enum>
          </property>
          <property name="defaultDropAction">
           <enum>Qt::IgnoreAction</enum>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::ExtendedSelection</enum>
          </property>
          <property name="iconSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
          <column>
           <property name="text">
            <string>File Tree</string>
           </property>
          </column>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QPushButton" name="ClearFileTreeButton">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Clear list</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="ExportOgreButton">
            <property name="text">
             <string>Export Ogre</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="frame_2">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QTabWidget" name="tabWidget">
          <property name="currentIndex">
           <number>1</number>
          </property>
          <widget class="QWidget" name="SkeletonTab">
           <attribute name="title">
            <string>Skeleton</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="BonesTreeWidget" name="skeletonTreeWidget">
              <property name="mouseTracking">
               <bool>false</bool>
              </property>
              <property name="focusPolicy">
               <enum>Qt::WheelFocus</enum>
              </property>
              <property name="contextMenuPolicy">
               <enum>Qt::CustomContextMenu</enum>
              </property>
              <property name="dragEnabled">
               <bool>false</bool>
              </property>
              <property name="dragDropMode">
               <enum>QAbstractItemView::InternalMove</enum>
              </property>
              <property name="defaultDropAction">
               <enum>Qt::MoveAction</enum>
              </property>
              <property name="selectionMode">
               <enum>QAbstractItemView::ExtendedSelection</enum>
              </property>
              <property name="selectionBehavior">
               <enum>QAbstractItemView::SelectItems</enum>
              </property>
              <property name="indentation">
               <number>10</number>
              </property>
              <attribute name="headerMinimumSectionSize">
               <number>100</number>
              </attribute>
              <column>
               <property name="text">
                <string>Bone Tree</string>
               </property>
              </column>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_10">
              <item>
               <spacer name="horizontalSpacer">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QPushButton" name="mergeSkeletonButton">
                <property name="text">
                 <string>Merge</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="MeshTab">
           <attribute name="title">
            <string>Mesh</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_9">
            <item>
             <widget class="QGroupBox" name="transformGroupBox">
              <property name="title">
               <string>Transform</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout">
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_5">
                 <item>
                  <widget class="QLabel" name="label">
                   <property name="text">
                    <string>Position</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QDoubleSpinBox" name="posXSpinBox">
                   <property name="prefix">
                    <string>X: </string>
                   </property>
                   <property name="decimals">
                    <number>5</number>
                   </property>
                   <property name="minimum">
                    <double>-100.000000000000000</double>
                   </property>
                   <property name="maximum">
                    <double>100.000000000000000</double>
                   </property>
                   <property name="singleStep">
                    <double>0.010000000000000</double>
                   </property>
                   <property name="value">
                    <double>0.000000000000000</double>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QDoubleSpinBox" name="posYSpinBox">
                   <property name="prefix">
                    <string>Y: </string>
                   </property>
                   <property name="decimals">
                    <number>5</number>
                   </property>
                   <property name="minimum">
                    <double>-100.000000000000000</double>
                   </property>
                   <property name="maximum">
                    <double>100.000000000000000</double>
                   </property>
                   <property name="singleStep">
                    <double>0.010000000000000</double>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QDoubleSpinBox" name="posZSpinBox">
                   <property name="prefix">
                    <string>Z: </string>
                   </property>
                   <property name="decimals">
                    <number>5</number>
                   </property>
                   <property name="minimum">
                    <double>-100.000000000000000</double>
                   </property>
                   <property name="maximum">
                    <double>100.000000000000000</double>
                   </property>
                   <property name="singleStep">
                    <double>0.010000000000000</double>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout">
                 <item>
                  <spacer name="horizontalSpacer_2">
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>40</width>
                     <height>20</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                 <item>
                  <widget class="QPushButton" name="applyTransformButton">
                   <property name="text">
                    <string>Apply</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="groupBox">
              <property name="title">
               <string>Bone assigment</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_5">
               <item>
                <widget class="QTableWidget" name="vertexBoneAssignmentMeshTable">
                 <property name="columnCount">
                  <number>3</number>
                 </property>
                 <attribute name="verticalHeaderVisible">
                  <bool>false</bool>
                 </attribute>
                 <column>
                  <property name="text">
                   <string>Bone name</string>
                  </property>
                 </column>
                 <column>
                  <property name="text">
                   <string>Bone handle</string>
                  </property>
                 </column>
                 <column>
                  <property name="text">
                   <string>Vertices count</string>
                  </property>
                 </column>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="TextureTab">
           <attribute name="title">
            <string>Texture</string>
           </attribute>
           <layout class="QHBoxLayout" name="horizontalLayout_9">
            <item>
             <widget class="QFrame" name="frame_8">
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_10">
               <item>
                <widget class="QGraphicsView" name="ddsTextureView">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="minimumSize">
                  <size>
                   <width>256</width>
                   <height>256</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>1024</width>
                   <height>1024</height>
                  </size>
                 </property>
                 <property name="sizeAdjustPolicy">
                  <enum>QAbstractScrollArea::AdjustToContents</enum>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QFrame" name="frame_6">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="frameShape">
                  <enum>QFrame::StyledPanel</enum>
                 </property>
                 <property name="frameShadow">
                  <enum>QFrame::Raised</enum>
                 </property>
                 <layout class="QHBoxLayout" name="horizontalLayout_8">
                  <item>
                   <widget class="QPushButton" name="LoadTextureButton">
                    <property name="text">
                     <string>Load from file...</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPushButton" name="SaveTextureButton">
                    <property name="text">
                     <string>Save to file...</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QFrame" name="frame_5">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <widget class="QTreeWidget" name="AnimationTree">
        <column>
         <property name="text">
          <string>Animation List</string>
         </property>
        </column>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="ClearAnimationListButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Clear list</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>Playback Controls</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QPushButton" name="PlayButton">
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="xenoviewer.qrc">
               <normaloff>:/icons/play.png</normaloff>:/icons/play.png</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="PauseButton">
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="xenoviewer.qrc">
               <normaloff>:/icons/pause.png</normaloff>:/icons/pause.png</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="StopButton">
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="xenoviewer.qrc">
               <normaloff>:/icons/stop.png</normaloff>:/icons/stop.png</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="PreviousFrameButton">
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="xenoviewer.qrc">
               <normaloff>:/icons/previous_frame.png</normaloff>:/icons/previous_frame.png</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="NextFrameButton">
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="xenoviewer.qrc">
               <normaloff>:/icons/next_frame.png</normaloff>:/icons/next_frame.png</iconset>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QCheckBox" name="PlaybackLoopCheck">
           <property name="text">
            <string>Playback Animation Loop</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FileTreeWidget</class>
   <extends>QTreeWidget</extends>
   <header>FileTreeWidget.h</header>
  </customwidget>
  <customwidget>
   <class>BonesTreeWidget</class>
   <extends>QTreeWidget</extends>
   <header>BonesTreeWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="xenoviewer.qrc"/>
 </resources>
 <connections/>
</ui>
